tools/etc/frr/support_bundle_commands.conf

```diff
@@ -153,3 +153,41 @@ show ip pim state
 show ip pim statistics
 show ip pim rpf
 CMD_LIST_END
+
+# OSPFv3 Support Bundle Command List
+PROC_NAME:ospf6
+CMD_LIST_START
+show ipv6 ospf6
+do show ipv6 ospf6 border-routers
+do show ipv6 ospf6 border-routers detail
+show ipv6 ospf6 database
+show ipv6 ospf6 database detail
+show ipv6 ospf6 database dump
+show ipv6 ospf6 database internal
+show ipv6 ospf6 database router detail
+show ipv6 ospf6 database network detail
+show ipv6 ospf6 database inter-prefix detail
+show ipv6 ospf6 database inter-router detail
+show ipv6 ospf6 database intra-prefix detail
+show ipv6 ospf6 database link detail
+show ipv6 ospf6 database as-external detail
+show ipv6 ospf6 database self-originate detail
+show ipv6 ospf6 database type-7 detail
+show ipv6 ospf6 interface
+show ipv6 ospf6 interface prefix
+show ipv6 ospf6 interface traffic
+show ipv6 ospf6 linkstate detail
+show ipv6 ospf6 neighbor
+show ipv6 ospf6 neighbor drchoice
+show ipv6 ospf6 neighbor detail
+show ipv6 ospf6 redistribute
+show ipv6 ospf6 route
+show ipv6 ospf6 route external-1
+show ipv6 ospf6 route external-2
+show ipv6 ospf6 route inter-area
+show ipv6 ospf6 route intra-area
+show ipv6 ospf6 route detail
```

Comment:
we need both route and route detail and route summary?

Reply:
I have kept both considering that it is easier to first look into the route output and then detail if required.